db/schema.rb

```diff
@@ -106,6 +106,19 @@
   add_index "diaries", ["cached_slug"], name: "index_diaries_on_cached_slug", using: :btree
   add_index "diaries", ["owner_id"], name: "index_diaries_on_owner_id", using: :btree
 
+  create_table "bookmarks", force: :cascade do |t|
+    t.string   "title",             limit: 160
+    t.string   "cached_slug",       limit: 165
+    t.integer  "owner_id",          limit: 4
+    t.string   "link",              limit: 1024,        null: false
+    t.string   "lang",              limit: 2, null: false
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  add_index "bookmarks", ["cached_slug"], name: "index_bookmarks_on_cached_slug", using: :btree
+  add_index "bookmarks", ["owner_id"], name: "index_bookmarks_on_owner_id", using: :btree
```

Comment:
I don't see the migration for this. Did you forget to include it in the commits?

Reply:
I did not write it. Now it's done :)